debian/control

```diff
@@ -2,13 +2,12 @@ Source: ralph
 Section: python
 Priority: extra
 Maintainer: pylabs@allegrogroup.com
-Build-Depends: debhelper (>= 9), python
```

Comment:
IMO git, python3-virtualenv, python3-dev and dh-virtualenv should not be required to install deb package - they are required only to build package